azure-pipelines.yml

```diff
@@ -275,6 +279,11 @@ jobs:
     displayName: 'Delete CodeSignSummary files'
     inputs:
       contents: '**\CodeSignSummary-*.md'
+  - task: ManifestGeneratorTask@0
+    displayName: 'SBOM Generation Task - SiteExtension'
+    inputs:
+      BuildDropPath: '$(Build.ArtifactStagingDirectory)\SiteExtension'
```

Comment:
Does this leave residual files in the drop path?

Reply:
It creates a `_manifest` folder in the drop path of each of the `ManifestGeneratorTask@0` as you can see if you download the artifacts.